global.d.ts

```diff
@@ -1,5 +1,20 @@
 declare module '@pixi/settings';
 
+declare type InteractionTrackingData = any;
+
+declare interface IHitArea {
+    contains(x: number, y: number): boolean;
+}
+
+declare interface InteractiveTarget {
+    interactive: boolean;
+    interactiveChildren: boolean;
+    hitArea: IHitArea;
+    cursor: string;
```

Comment:
maybe add something from here? https://github.com/frenic/csstype/blob/019f3524660f76764cded3fd4e429a353ef3464b/index.js.flow#L4359

Reply:
Good idea @lekzd

Reply:
Don't know how i didn't see this. Added it now.